ipu-plugin/pkg/ipuplugin/bridgeport_test.go

```diff
@@ -107,7 +107,7 @@ var _ = Describe("bridgeport", Serial, func() {
 					},
 				}
 				fakePortBridgeInfo := &types.BridgePortInfo{
-					fakePort, "fakePort1",
+					fakePort, 10,
```

Comment:
What is 10? Maybe comment needed to explain the magic number?

Reply:
It is an integer for the Port's IntefaceId as defined in the struct type. This is for test code...previously it was string->"fakePort1"

Reply:
Is it a random integer? Can it be 11 or 12?